### DISCUSSION TITLE
Simple Neural Network

### DISCUSSION THREAD

**Discussion by tekelsey** [jerryjee120/Neural-Network#3] [General]
Jerry, would you be interested to help figure out a very simple example of a neural network?

If so I am open to helping and it could be included as a tutorial in app.

If you can help me understand the math (I have PhD) I can help make it understandable (I am professor and have written 20 books)

You can connect with me here:
http://linkedin.com/in/tekelsey

**Comment by jerryjee120** [comment]
Hello Tekelsey, I have connected you on Linkin(https://www.linkedin.com/in/jerry-jee-8235b6218/). 
You can tell me the details of your thoughts by sending a linkin message or email (jerryjee120@gmail.com).

Thanks!
